### PR TITLE
FB - Link OVHcloud domain to a Google Site

### PR DESCRIPTION
**Pull request by fabienOVH**

### CODE REVIEW

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: Comment connecter un nom de domaine OVHcloud à un Google Site
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
title: "Comment connecter un nom de domaine OVHcloud à un Google Site"
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+excerpt: Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour le connecter à un Google Site
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
excerpt: "Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour le connecter à un Google Site"
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+updated: 2024-09-20
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
updated: 2024-09-25
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+---
+
+## Objectif
+
+Vous possédez un nom de domaine chez OVHcloud et vous souhaitez le connecter à un Google Site. Vous trouverez dans ce guide les étapes de préparation et de configuration de votre zone DNS OVHcloud pour permettre la configuration de votre Google Site.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
Vous possédez un nom de domaine chez OVHcloud et vous souhaitez le connecter à un Google Site. Retrouvez dans ce guide les étapes de préparation et de configuration de votre zone DNS OVHcloud pour permettre la configuration de votre Google Site.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+> - L’assistance Google Site n’a pas accès aux paramètres de votre nom de domaine OVHcloud et ne peut donc pas vous conseiller sur les informations que vous devrez lui fournir.
+> - OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.<br><br> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](/links/partner) et/ou de contacter l'éditeur du service si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#gofurther) de ce guide.
+>
+
+## Prérequis
+
+- Être connecté à votre [espace client OVHcloud](/links/manager){.external}.
+- Disposer d'un [nom de domaine](/links/web/domains){.external} enregistré chez OVHcloud.
+- Disposer des [autorisations appropriées pour gérer](/pages/account_and_service_management/account_information/managing_contacts) le nom de domaine depuis votre [espace client OVHcloud](/links/manager){.external}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
- Disposer des [autorisations appropriées](/pages/account_and_service_management/account_information/managing_contacts) pour gérer le nom de domaine depuis votre [espace client OVHcloud](/links/manager){.external}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+- Disposer d’un Google Site et en être le propriétaire.
+
+## En pratique
+
+Avant de suivre les étapes de ce guide, nous vous conseillons de vous familiariser avec la configuration d'une zone DNS à l'aide de notre guide « [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit) ».
+
+> [!warning]
+>
+> Votre zone DNS est potentiellement déjà préconfigurée ou liée à un hébergement. Nous allons voir comment identifier chaque enregistrement DNS nécessaire à la connexion avec votre Google Site. Certains devront être supprimés pour éviter de rentrer en conflit avec des enregistrements DNS requis dans cette configuration. D'autres seront simplement à modifier ou à créer. Pour une meilleure compréhension, nous utiliserons le nom de domaine « **mydomain.ovh** » comme exemple. Remplacez celui-ci par votre nom de domaine lors de la configuration.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
> Votre zone DNS est potentiellement déjà préconfigurée ou liée à un hébergement Web. Nous allons voir comment identifier chaque enregistrement DNS nécessaire à la connexion avec votre Google Site. Certains devront être supprimés pour éviter de rentrer en conflit avec des enregistrements DNS requis dans cette configuration. D'autres devront juste être modifiés ou créés. Pour une meilleure compréhension, nous utiliserons le nom de domaine « **mydomain.ovh** » comme exemple. Remplacez celui-ci par votre nom de domaine lors de la configuration.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+### 1. Configurer votre Google Site
+
+> [!warning]
+>
+> Seul le propriétaire d'un Google Site peut le connecter à un domaine. Découvrez comment [modifier le propriétaire du site](https://support.google.com/sites/answer/97934){.external}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
> Seul le propriétaire d'un Google Site peut le connecter à un nom de domaine. Si besoin, découvrez comment [modifier le propriétaire du Google site](https://support.google.com/sites/answer/97934){.external}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+Lorsque vous utilisez un Google Site avec un nom de domaine OVHcloud, vous devez d'abord préparer votre hébergement en suivant les instructions de la section **Configurer un domaine personnalisé** depuis [**cette page du support Google**](https://support.google.com/sites/answer/9068867?hl=fr#zippy=).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
Lorsque vous utilisez un Google Site avec un nom de domaine OVHcloud, préparez d'abord votre hébergement en suivant les instructions de la section **Configurer un domaine personnalisé** depuis [**cette page du support Google**](https://support.google.com/sites/answer/9068867?hl=fr#zippy=){.external}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+### 2. Configurer vos enregistrements DNS sur votre compte OVHcloud
+
+
+Connectez-vous à votre [espace client OVHcloud](/links/manager){.external} dans la section `Web Cloud`{.action}. Cliquez sur `Noms de domaine`{.action}, puis choisissez le nom de domaine concerné. Positionnez-vous enfin sur l'onglet `Zone DNS`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
Connectez-vous à votre [espace client OVHcloud](/links/manager){.external}, puis rendez-vous dans la section `Web Cloud`{.action}. Cliquez sur `Noms de domaine`{.action} dans la colonne de gauche, puis choisissez le nom de domaine concerné. Positionnez-vous enfin sur l'onglet `Zone DNS`{.action}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+
+Le tableau qui s'affiche liste l'ensemble des enregistrements DNS du nom de domaine sélectionné.
+
+![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/tab-mydomain-anycast.png){.thumbnail}
+
+Chaque enregistrement DNS peut être modifié en cliquant sur le bouton `...`{.action} à droite de la ligne du tableau concernée puis en cliquant sur `Modifier l'entrée`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
Chaque enregistrement DNS peut être modifié en cliquant sur le bouton `...`{.action} à droite de la ligne du tableau concernée, puis en cliquant sur `Modifier l'entrée`{.action}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+Suivez les étapes dans l'ordre en parcourant les onglets suivants :
+
+> [!tabs]
+> **Etape 1**
+>> **Enregistrement A**<br><br>
+>> Pour identifier les enregistrements « A » existants, cliquez sur le menu des filtres en haut du tableau d'enregistrements DNS et sélectionnez `A`.<br>
+>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-a.png){.thumbnail}
+>>
+>> - Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau qui correspond à votre nom de domaine seul, sans sous-domaine (exemple : `mydomain.ovh.`), puis cliquez sur `Modifier l'entrée`{.action}.<br>
+>> - Si un enregistrement pour le sous-domaine « www » est présent (exemple : `www.mydomain.ovh.`), vous devez le supprimer pour qu'il ne rentre pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre nom de domaine seul avec le sous-domaine « www »  puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> - Si un enregistrement pour le sous-domaine « www » est présent (exemple : `www.mydomain.ovh.`), supprimez-le pour qu'il ne rentre pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre sous-domaine en « www »,  puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+>> - Si vous n'avez pas d'enregistrement « A » existant, cliquez sur le bouton `Ajouter une entrée`{.action} en haut à droite de votre écran et sélectionnez le « Champ de pointage » `A`{.action}<br><br>
+>> Vous devez créer quatre enregistrements de type « A » successivement afin de renseigner les quatre adresses IPv4 relatives à Google Site.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> Créez 4 enregistrements de type « A » successivement afin de renseigner les 4 adresses IPv4 relatives à Google Site. 
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+>> Laissez le champ **Sous-domaine** vide et saisissez la première adresse IPv4 de Google Site `216.239.32.21` dans le champ **Cible**.
+>> Cliquez sur `Suivant`{.action} et validez votre enregistrement « A ». Répétez l'opération pour les trois autres adresse IPv4 `216.239.34.21`, `216.239.36.21` et `216.239.38.21`, puis passez à l'étape 2. La valeur de ces adresses IP étant susceptibles de changer, vérifiez dans la documentation officielle [la valeur des enregistrements A](https://support.google.com/a/answer/2579934?hl=fr&ref_topic=2721296&sjid=10373374977980680534-EU){.external}.
+> **Etape 2**
+>> **Enregistrement AAAA**<br><br>
+>> Pour identifier les enregistrements « AAAA » existants, cliquez sur le menu des filtres en haut du tableau d'enregistrements DNS et sélectionnez `AAAA`.<br>
+>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-aaaa.png){.thumbnail}
+>>
+>> - Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau qui correspond à votre nom de domaine seul, sans sous-domaine (exemple : `mydomain.ovh.`), puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> - Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre nom de domaine seul, sans sous-domaine (exemple : `mydomain.ovh.`), puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+>> - Si un enregistrement pour le sous-domaine « www » est présent (exemple : `www.mydomain.ovh.`), supprimez-le également pour qu'il ne rentre pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre nom de domaine avec le sous-domaine « www » puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> - Si un enregistrement pour le sous-domaine « www » est présent (exemple : `www.mydomain.ovh.`), supprimez-le également pour qu'il ne rentre pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre sous-domaine en « www » puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+>> - Si vous n'avez pas d'enregistrement « AAAA » existant, passez à l'étape 3.
+> **Etape 3**
+>> **Enregistrement TXT**<br><br>
+>>  Pour identifier les enregistrements « TXT » existants, cliquez sur le menu des filtres en haut du tableau d'enregistrements DNS et sélectionnez `TXT`.<br>
+>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-txt.png){.thumbnail}
+>>
+>> - Si des enregistrements « TXT » sont présents pour le nom de domaine seul (exemple : `mydomain.ovh.`) et pour son sous-domaine en « www » (exemple : `www.mydomain.ovh.`), vous devez les supprimer pour qu'ils ne rentrent pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre nom de domaine seul avec le sous-domaine « www »  puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> - Si des enregistrements « TXT » sont présents pour le nom de domaine seul (exemple : `mydomain.ovh.`) et pour son sous-domaine en « www » (exemple : `www.mydomain.ovh.`), supprimez-les pour qu'ils ne rentrent pas en conflit avec l'enregistrement CNAME que vous allez renseigner à l'étape 4. Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre  sous-domaine en « www »  puis cliquez sur `Supprimer l'entrée`{.action}.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+>> Complétez les champs **Sous-domaine** et **Cible** avec les informations présentes sur la page « [Valeurs des enregistrements TXT](https://support.google.com/a/answer/2716802?hl=fr&ref_topic=2716886&sjid=3052810298579211755-EU){.external} » de la documentation officielle. Généralement, la valeur du champ **Sous-domaine** est vide, et celle du champ **Cible** est de type `google-site-verification=XXXXXXXXXXXX`.<br>
+>> Cliquez sur `Suivant`{.action} pour valider votre enregistrement « TXT » et passez à l'étape 4.
+> **Etape 4**
+>> **Enregistrement CNAME**<br><br>
+>>  Pour identifier les enregistrements « CNAME » existants, cliquez sur le menu des filtres en haut du tableau d'enregistrements DNS et sélectionnez `CNAME`.<br>
+>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-cname.png){.thumbnail}
+>>
+>> - Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre sous-domaine en « www.» (exemple : `mydomain.ovh.`) puis cliquez sur `Modifier l'entrée`{.action}.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
>> - Cliquez sur le bouton `...`{.action} à droite de la ligne du tableau correspondant à votre sous-domaine en « www » (exemple : `www.mydomain.ovh.`) puis cliquez sur `Modifier l'entrée`{.action}.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+>> Complétez le champ **Sous-domaine** avec la valeur `www` et saisissez `ghs.googlehosted.com.` dans le champ **Cible**. Ces valeurs étant amenées à changer, vérifiez-les sur la page « [Valeurs des enregistrements CNAME](https://support.google.com/a/answer/112038?sjid=3052810298579211755-EU){.external} » de la documentation officielle<br>
+>> Cliquez sur `Suivant`{.action} pour valider votre enregistrement « CNAME ».
+
+La zone DNS est maintenant configurée pour faire le lien avec votre Google Site.
+
+> [!primary]
+>
+> La vérification de votre nom de domaine peut prendre jusqu’à 48 heures.
+
+Si vous utilisez une offre e-mail OVHcloud ou que vous prévoyez de souscrire à l'une de [nos offres e-mail](/links/web/emails), vous devez également préparer vote zone DNS en conséquence. Consultez notre guide sur la [configuration d’un enregistrement MX](/pages/web_cloud/domains/dns_zone_mx).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]:
```suggestion
Si vous utilisez une offre e-mail OVHcloud ou que vous prévoyez de souscrire à l'une de [nos offres e-mail](/links/web/emails), préparez vote zone DNS en conséquence. Consultez notre guide sur la [configuration d’un enregistrement MX](/pages/web_cloud/domains/dns_zone_mx).
```

**Review by Kazarus4 — COMMENTED** [review]
FRFR done for me, need to create the new line into the index file before merge.

**Comment by Y0Coss** [comment]
Hello @fabienOVH , the FR version is now ready for prod. Could you please provide us with an EN version?

**Comment by fabienOVH** [comment]
> Hello @fabienOVH , the FR version is now ready for prod. Could you please provide us with an EN version?

Hello @Y0Coss, I've just pushed the EN version

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
+---
+title: "How to connect an OVHcloud domain name to a Google Site"
+excerpt: "Prepare and configure your OVHcloud domain name’s DNS zone to connect to a Google Site"
+updated: 2024-09-25
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
updated: 2024-09-26
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+>
+
+## Requirements
+
+- You must be logged in to your [OVHcloud Control Panel](/links/manager).
+- A [domain name](/links/web/domains) registered with OVHcloud.
+- You must have the [appropriate permissions](/pages/account_and_service_management/account_information/managing_contacts) to manage the domain name from your [OVHcloud Control Panel](/links/manager).
+- You must own and own a Google Site.
+
+## In practice
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
## Instructions
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+> [!warning]
+>
+> Your DNS zone is potentially already preconfigured or linked to a web hosting plan. We will look at how to identify each DNS record required to log in to your Google Site. Some records will need to be deleted to avoid conflicts with DNS records required in this configuration. Others will just have to be modified or created. For a better understanding, we will use the domain name **mydomain.ovh** as an example. Replace it with your domain name during configuration.
+
+### Step 1 - Configure your Google Site
+
+> [!warning]
+>
+> Only the owner of a Google Site can connect it to a domain name. If necessary, find out how to [change the owner of the Google site](https://support.google.com/sites/answer/97934).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md:38]: The link redirect to a FR page. Please could you change the FR link by the EN link ?

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+
+### Step 2 - Configure your DNS records on your OVHcloud account
+
+Log in to your [OVHcloud Control Panel](/links/manager), then go to the `Web Cloud`{.action} section. Click `Domain names`{.action} in the left-hand column, then choose the domain name concerned. Go to the `DNS Zone`{.action} tab.
+
+The table that opens lists all of the DNS records for the selected domain name.
+
+![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/tab-mydomain-anycast.png){.thumbnail}
+
+Each DNS record can be modified by clicking the '...`{.action}' button to the right of the table row concerned, then clicking `Modify record`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
Each DNS record can be modified by clicking the `...`{.action} button to the right of the table row concerned, then clicking `Modify record`{.action}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+
+Follow the steps in order by going through the following tabs:
+
+> [!tabs]
+> **Step 1**
+>> **Record A**<br><br>
+>> To identify existing A records, click the filter menu at the top of the DNS records table and select `A`.
+>>
+>>![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-a.png){.thumbnail}
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-a.png){.thumbnail}
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+>> - If you do not have an existing A record, click the `Add record`{.action} button in the top right-hand corner of your screen and select the `A`{.action}<br><br>
+>> Create 4 A records in succession to enter the 4 IPv4 addresses related to Google Site.
+>> Leave the **Sub-domain** field empty and enter the first IPv4 address of Google Site `216.239.32.21` in the **Target** field.
+>> Click `Next`{.action} and confirm your A record. Repeat for the other three IPv4 addresses `216.239.34.21`, `216.239.36.21`, and `216.239.38.21`, then go to step 2. Since the values of these IP addresses may change, check the official documentation [the value of A records](https://support.google.com/a/answer/2579934?hl=en&ref_topic=2721296&sjid=10373374977980680534-EU).
+>>
+> **Step 2**
+>> **AAAA record**<br><br>
+>> To identify the existing "AAAA" records, click on the filter menu at the top of the DNS records table and select `AAAA`.
+>>
+>>![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-aaaa.png){.thumbnail}
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-aaaa.png){.thumbnail}
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 24]
```diff
@@ -0,0 +1,124 @@
...
+>>
+>> - Click the `...`{.action} button to the right of the table row for your single domain name, without a subdomain (e.g. `mydomain.ovh.`), then click `Delete record`{.action}.<br>
+>> - If a record for the “www” subdomain is present (e.g.: `www.mydomain.ovh.`), delete it as well so that it does not conflict with the CNAME record you enter in step 4. Click the `...`{.action} button to the right of the table row corresponding to your subdomain in “www” , then click `Delete record`{.action}.<br>
+>> - If you do not have an existing AAAA record, proceed to step 3.
+>>
+> **Step 3**
+>> **TXT record**<br><br>
+>> To identify existing “TXT” records, click the filter menu at the top of the DNS records table and select `TXT`.
+>>
+>>![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-txt.png){.thumbnail}
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-txt.png){.thumbnail}
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 25]
```diff
@@ -0,0 +1,124 @@
...
+>> - You must create a record of type “TXT”. Click on the `Add record`{.action} button in the top right-hand corner of your screen and select the `TXT`{.action} “Pointing field”.
+>> Enter the **Sub-domain** and **Target** fields with the information provided on the page “[TXT record values](https://support.google.com/a/answer/2716802?hl=en&ref_topic=2716886&sjid=3052810298579211755-EU)” in the official documentation. Typically, the **Sub-domain** field is empty, and the **Target** field is of type `google-site-verification=XXXXXXXXXXXX`.<br>
+>> Click `Next`{.action} to validate your TXT record and proceed to step 4.
+>>
+> **Step 4**
+>> **CNAME record**<br><br>
+>> To identify existing CNAME records, click on the filter menu at the top of the DNS records table and select `CNAME`.
+>>
+>>![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-cname.png){.thumbnail}
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-cname.png){.thumbnail}
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md [hunk 26]
```diff
@@ -0,0 +1,124 @@
...
+[Edit an OVHcloud DNS zone](/pages/web_cloud/domains/dns_zone_edit)
+
+To delegate the management of your domain name to another OVHcloud customer account, follow the guide on [Managing contacts for OVHcloud services](/pages/account_and_service_management/account_information/managing_contacts).
+
+For specialized services (SEO, development, etc.), contact the [OVHcloud partners](/links/partner).
+
+If you would like assistance using and configuring your OVHcloud solutions, please refer to our [support offers](/links/support).
+
+Join our [community of users](/links/community).
+
```
Comment (Kazarus4) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md]:
```suggestion
```

**Review by Kazarus4 — COMMENTED** [review]
EN review done for me

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md (hunk 20 above)
Comment (fabienOVH) [reply pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md:38]: Really ? For me it redirects to the official english page (the english language is picked up in the bottom of the page in the list). Are you sure your browser is not forcing the french translation ?)

Comment (Y0Coss) [reply pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md:38]: I agree with Fabien. My browser language is set to English, the link leads to a page in English.

**Pull request merged**